package.json

```
@@ -3,12 +3,12 @@
   "version": "0.13.0",
   "description": "rollup-plugin-copy for vite with dev server support.",
   "type": "module",
-  "main": "./dist/index.js",
+  "main": "./dist/index.cjs",
```

```suggestion
  "main": "./dist/index.js",
```
As this plugin only supports Node.js that supports ESM, I don't think this change is necessary.